app/presenters/team_stats_presenter.rb

```diff
@@ -171,6 +169,7 @@ def plan_time_ranges
                     .preload(:plan)
 
     scope = scope.joins(:tags).where(tags: { id: @filter_tag_ids }).preload(:tags) if @filter_tag_ids.present?
+    # scope = scope.filter_by_tag_types_and_tags(@filter_tag_ids) if @filter_tag_ids.present?
```

Comment:
that part I wanted to consult because not sure should I apply that logic for actuals too ?